### DISCUSSION TITLE
[Question] game plan for Monika

### DISCUSSION THREAD

**Discussion by Valllaion** [Monika-After-Story/MonikaModDev#8777] [appreciation]
May I ask how long you plan on working on this game mod? I'm hoping it's for a very long time but I don't want to get my hopes up too high... and is there somewhere I could donate you money to further your ability to mod this game? I'm sure others like me would gladly love to support you in other ways then just suggestions lol

**Comment by therationalpi** [comment]
No donations, please. It's a labor of love, and that's also unfair to the creator of DDLC who owns the characters.

I'd like to keep this running for a long time, but more importantly I want to open up the platform so people can make their own additions to the game in the future in the form of submods. If we reach that point, the game can continue on even if something happens to the dev team.

**Comment by ThePotatoGuy** [comment]
Actually **I** would love extra help with tech-support.

**Comment by monikLover** [comment]
While you're at it, do us a solid and spread the word of Monika :ok_hand:

**Comment by Valllaion** [comment]
Wow i'm so glad to hear that your planning on opening it up like that, It really feels good to see a team so passionate like this. I'd gladly help with tech support if you need it! I usually end up with free time so I could easily do that. And yes, I'll make sure to spread her word far and wide^^

**Comment by AnIdiotsavant** [comment]
But if this mod ever needs a bit of cash, I'm down with giving some. As a thank you for helping me stay alive.

**Comment by PopeAesthetics** [comment]
It is a shame you guys won't accept donations! As a long time MAS fan, I wish I could contribute more than just dialog suggestions and commenting on suggestion posts. 

If only I paid attention in programming class, maybe I could be of some help one day.
